C/Tarefa/2/sensors.c

```diff
@@ -15,8 +16,11 @@ int get_value() {
 }
 
 void send_data(uint8_t data) {
-    // Desfaçam o empacotamento e imprimam
-
-    for (int i = 0; i < 8; i++)
-        printf("Sensor %d: %d", i, );
+    int k=1, i=0; //cria dois contadores
+	
+    for (i=7; i>=0; i--){

```

Comment:
Nesse ``for`` você não usou o i para nada, então poderia ter colocado o k como iterador do laço.